hardware/products/ucm/intro.md

```diff
@@ -230,7 +265,7 @@ The peripherals found on each module may vary.
 | 197           | JTAG TCK (SWCLK)              |
 | 198           | GND                           |
 | 199           | JTAG TMS (SWDIO)              |
-| 200           | Reserved                      |
+| 200           | Indicator D                   |
 
 ## UCM Standard - Module Dimesions
```

Comment:
Wasn't it wanted to take these out?

Reply:
I do not understand, take what out? The Indicators were being added and the original Ethernet LEDs were to be changed to the Indicators as well and I2S was to be removed as well. Bootstrap A and MS1 were to be swapped which is what was done.

Reply:
This is on the line for dimensions, not indicators.